Source/Android/jni/MainAndroid.cpp

```diff
@@ -146,14 +146,14 @@ void Host_TitleChanged()
 {
 }
 
-static bool MsgAlert(const char* caption, const char* text, bool yes_no, Common::MsgType /*style*/)
+static bool MsgAlert(const char* caption, const char* text, bool yes_no, Common::MsgType style)
 {
   JNIEnv* env = IDCache::GetEnvForThread();
 
   // Execute the Java method.
   jboolean result = env->CallStaticBooleanMethod(
       IDCache::GetNativeLibraryClass(), IDCache::GetDisplayAlertMsg(), ToJString(env, caption),
-      ToJString(env, text), yes_no ? JNI_TRUE : JNI_FALSE);
+      ToJString(env, text), yes_no ? JNI_TRUE : JNI_FALSE, style == Common::MsgType::Warning);
```

Comment:
I don't know if having the style affect the behavior of the panic alert is a good idea or not. It would be better if this commit could be split into another PR so that the merge of the other fixes in this PR aren't blocked by this question being open.

Reply:
Actually nevermind lol we already do this in Qt. So I guess it's fine